### DISCUSSION TITLE
How to secure strimzi kafka connect endpoint over TLS?

### DISCUSSION THREAD

**Discussion by bharat15-ml** [strimzi/strimzi-kafka-operator#6102] [General]
Currently, have deployed connect cluster with 3 nodes and able to create mongodb source and sink connectors, by calling connect endpoint over http and it's working fine.

I wanted to secure connect endpoint over https. Please suggest with what configuration changes and how can we secure connect end point over TLS so that it can be called over https instead of http?

**Comment by scholzj** [comment]
This is currently not supported. But you can check the proposal about this: https://github.com/strimzi/proposals/pull/45

**Reply by bharat15-ml** [reply]
Thanks for the quick update!! Would like to know on expected delivery timelines for this feature, So accordingly can plan for production deployment.

**Reply by scholzj** [reply]
Well, the proposal needs to be approved first and then it needs to be implemented. I do not think anyone can give you any fixed timelines.
